src/main/webapp/WEB-INF/web.xml

```diff
@@ -66,4 +66,15 @@
     <url-pattern>/register</url-pattern>
   </servlet-mapping>
 
+  <servlet>
```

Comment:
Oh, I just saw that there's a merge conflict in here, so do a `git pull origin master` and clear that up.